kafka/README.md

```diff
@@ -33,11 +51,14 @@ This object contains information about the operation representation in Kafka.
 
 ##### Fixed Fields
 
-Field Name | Type | Description
----|:---:|---
-<a name="operationBindingObjectGroupId"></a>`groupId` | [Schema Object][schemaObject] | Id of the consumer group.
-<a name="operationBindingObjectClientId"></a>`clientId` | [Schema Object][schemaObject] | Id of the consumer inside a consumer group.
-<a name="operationBindingObjectBindingVersion"></a>`bindingVersion` | string | The version of this binding. If omitted, "latest" MUST be assumed.
+Field Name | Type | Description | Applicability [default] | Constraints
+---|:---:|:---:|:---:|---
+<a name="operationBindingObjectGroupId"></a>`groupId` | [Schema Object][schemaObject] | Id of the consumer group. | OPTIONAL | -
+<a name="operationBindingObjectClientId"></a>`clientId` | [Schema Object][schemaObject] | Id of the consumer inside a consumer group. | OPTIONAL | -
+<a name="operationBindingObjectSchemaIdLocation"></a>`schemaIdLocation` | string | If a Schema Registry is used when performing this operation, tells where the id of schema is stored (e.g. `header` or `payload`). | OPTIONAL | MUST NOT be specified if `schemaRegistryUrl` is not specified at the Server level
+<a name="operationBindingObjectSchemaIdPayloadEncoding"></a>`schemaIdPayloadEncoding` | string | Number of bytes or vendor specific values when schema id is encoded in payload (e.g `confluent`/ `apicurio-legacy` / `apicurio-new`). | OPTIONAL | MUST NOT be specified if `schemaRegistryUrl` is not specified at the Server level
```

Comment:
My personal preference is towards describing the implementation (e.g. first four bytes, or first eight bytes) rather than use named vendors. I think that would be easier to maintain, as vendors are prone to change what they do or add new options. 

Plus this way we won't end up with multiple ways of describing the same thing (i.e. where multiple vendors have a similar approach)

Reply:
That's a good point. We could put the emphasis on the fact that putting the number of bytes is the preferred way of doing things. However, if people are not aware of internal implementations, they could fall back to just specifying the vendor encoding. 

Do you think we should introduce specific notations like `4-bytes` or `8-bytes` or let people put integers ?

Reply:
I don't have a really strong opinion... I can see arguments for both approaches. I think I lean slightly towards the descriptive enum approach, because it feels slightly more future-proof

Reply:
Propose to resolve it for the moment and keeping it simple.